mibs/vmware-sdwan/VELOCLOUD-MIB

```diff
@@ -0,0 +1,42 @@
+-------------------------------
```

Comment:
Please put this MIB in the vmware directory and add `mib_dir: vmware` on the OS.

Reply:
MIB directories are per-vendor, not per-os.